lgc/state/PipelineState.cpp

```diff
@@ -700,31 +728,20 @@ std::pair<const ResourceNode *, const ResourceNode *>
 PipelineState::findResourceNode(ResourceNodeType nodeType, unsigned descSet, unsigned binding) const {
```

Comment:
The same as above

Reply:
This has a comment in the right format, but the github UI hides it.